ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -47,6 +47,7 @@ tests:
     workflow: opendatahub-io-ci-image-mirror
 - as: opendatahub-operator-image-mirror
   postsubmit: true
+  skip_if_only_changed: ^\.github/|^docs/|\.[mM][dD]$|^.gitignore$|^golangci|^crd-ref-docs\.config|^OWNERS$|^PROJECT$|^LICENSE$|^OWNERS_ALIASES$
```

Comment:
Where the expression came from? I'm wondering what are `golangci` and `crd-ref-docs.config` parts are. Also dot is not escaped in `^.gitignore$` (it works, I doubt we will have other Xgitignore files but anyway)

Reply:
here the .github has escaped but not gitignore , this is weird :D

Reply:
> here the .github has escaped but not gitignore , this is weird :D

Exactly  :)

Reply:
but seems to work, so I guess it should be good 😄